src/option/config/datazoom/index.js

```
@@ -11,43 +11,46 @@
-        style.selectDataColor || selectedDataBackground.areaStyle.color;
-      dataZoom[0].fillerColor = style.middleFillerColor || fillerColor;
-      // 用户自定义手柄样式
-      if (style.handleStyle) {
-        dataZoom[0].handleStyle = Object.assign(dataZoom[0].handleStyle, style.handleStyle);
+  const self = iChartOption.dataZoom || {}
+  const dataZoom = toArray(self).map((item, index) => {
+    if (index === 0) {
+      const { show, position, start, end, startValue, endValue, style, height } = item;
+      const baseZoom = base()
+      if (show) {
+        end && (baseZoom.end = end);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)